packages/cli/templates/vue3/src/components/AssetSelector.vue

```diff
@@ -1,5 +1,5 @@
 <script setup lang="ts">
-import { defineProps, ref } from 'vue'
```

Comment:
![Screenshot 2023-03-30 at 10 09 58](https://user-images.githubusercontent.com/499898/228772267-29d99a12-6a12-40fa-8c99-b985df9734cc.png)

Didn't know that we don't have to import it!